### PR TITLE
Set proper winBaseUrl based on arch

### PR DESCRIPTION
**Pull request by aekeus**

### CODE REVIEW

File: app/updater.js [hunk 1]
```diff
@@ -25,6 +25,7 @@ const updateLogPath = path.join(app.getPath('userData'), 'updateLog.log')
 // in built mode console.log output is not emitted to the terminal
 // in prod mode we pipe to a file
 var debug = function (contents) {
+    console.log(contents)
```
Comment (bbondy) [inline app/updater.js]: nit: remove

File: app/updater.js [hunk 2]
```diff
@@ -47,13 +48,13 @@ exports.updateUrl = function (updates, platform, arch) {
...
   }
   platformBaseUrl = `${updates.baseUrl}/${Channel.channel()}/${version}/${platforms[platform]}`
-  console.log(platformBaseUrl)
   debug(`platformBaseUrl = ${platformBaseUrl}`)
   if (platform === 'darwin') {
     return platformBaseUrl
   } else {
-    debug(updates.winBaseUrl.replace('CHANNEL', Channel.channel()))
-    return updates.winBaseUrl.replace('CHANNEL', Channel.channel())
+    var windowsUpdateUrlWithArch = updates.winBaseUrl.replace('CHANNEL', Channel.channel()) + platforms[platform]
```
Comment (bbondy) [inline app/updater.js]: Variable name a bit off since linux will go in this clause too won't it? Or if not why not?

**Comment by bbondy** [comment]
++ otherwise

File: app/updater.js (hunk 2 above)
Comment (aekeus) [inline app/updater.js]: Yes, Linux will go here. Naming is a vestige from OSX + Windows only.

File: app/updater.js [hunk 3]
```diff
@@ -47,13 +48,17 @@ exports.updateUrl = function (updates, platform, arch) {
...
   }
   platformBaseUrl = `${updates.baseUrl}/${Channel.channel()}/${version}/${platforms[platform]}`
-  console.log(platformBaseUrl)
   debug(`platformBaseUrl = ${platformBaseUrl}`)
   if (platform === 'darwin') {
     return platformBaseUrl
   } else {
-    debug(updates.winBaseUrl.replace('CHANNEL', Channel.channel()))
-    return updates.winBaseUrl.replace('CHANNEL', Channel.channel())
+    if (platform.match(/^win32/)) {
```
Comment (bbondy) [inline app/updater.js]: is there a possibility for something suffixed to win32?

Comment (bbondy) [inline app/updater.js]: Also just a nit but else if block instead of if/else nested in else.

Comment (bbondy) [inline app/updater.js]: merging as is though

**Pull request merged**